slf4j-ext/src/test/java/org/slf4j/dummyExt/XLoggerTest.java

```diff
@@ -107,6 +107,9 @@ public void testThrowing() {
     assertEquals(t, logger.throwing(t));
     assertEquals(t, logger.throwing(XLogger.Level.DEBUG,t));
     assertEquals(2, listAppender.list.size());
+    if (listAppender.list.get(0).getThrowableStrRep() == null) {
```

Comment:
Its been some time since i did that but i believe it protects against possible null pointer that eclipse noted.

--- Original Message ---

From: "Matt Bishop" <notifications@github.com>
Sent: March 16, 2015 10:43 AM
To: "qos-ch/slf4j" <slf4j@noreply.github.com>
Cc: "Jeremy Landis" <jeremylandis@hotmail.com>
Subject: Re: [slf4j] Updating builds (#88)

> @@ -107,6 +107,9 @@ public void testThrowing() {
>      assertEquals(t, logger.throwing(t));
>      assertEquals(t, logger.throwing(XLogger.Level.DEBUG,t));
>      assertEquals(2, listAppender.list.size());
> +    if (listAppender.list.get(0).getThrowableStrRep() == null) {

Why this change?

---
Reply to this email directly or view it on GitHub:
https://github.com/qos-ch/slf4j/pull/88/files#r26489760